### PR TITLE
Added a viewport and changed to 𝖉𝖆𝖗𝖐 𝖙𝖍𝖊𝖒𝖊

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # Openbook
+The new kind of social network
```
Comment (Socerest2) [inline README.md]:
```suggestion
The new kind of social network
![Alt](https://github.com/Socerest2/openbook/blob/main/logos/openbook.png?raw=true "e")
```

**Review by Socerest2 — CHANGES_REQUESTED** [review]
If you want

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: I agree with that!

**Review by Socerest2 — APPROVED** [review]
yes

**Pull request merged**